pkg/features/kube_features.go

```diff
@@ -868,6 +868,12 @@ const (
 	//
 	// Enables specifying resources at pod-level.
 	PodLevelResources featuregate.Feature = "PodLevelResources"
+
+	// owner: @ffromani
+	// beta: v1.33
+	//
+	// Disable CPU Quota for pods which have exclusive CPUs allocated
```

Comment:
Can you update this to be a bit more explicit about what it does. Its removing it on _both_ the container level and the pod level, and explain the way we end up dealing with the init containers.

I think personally we could have removed the cpu limit for all containers that have exclusive CPUs as well (but not neccecary the pod level), but we can also do that in a separate PR/feature.

Reply:
Agreed about we should have a record of why we disable pod level/container level. We should also backlink the PRs on github and the issue for historical record

Reply:
note: we want to remove the FG on 1.33+4 releases anyway, so this has to be placed somewhere is gonna be kept once the FG is removed.